HLA-1070: New build badge for github actions, added installation docs (#1671)
spacetelescope/drizzlepac: README.md

## Conda (Recommended)

> ``Drizzlepac`` is installed when you install the ``stenv`` conda environment, formerly (astropy). Select your desired release through the [stenv readthedocs page](https://stenv.readthedocs.io/en/latest/getting_started.html) and follow the instructions for installation. 

Reviewer: ```suggestion
`Drizzlepac` is installed when you install the `stenv` conda environment (a replacement for `astroconda`). Select your desired release and follow the instructions on the [`stenv` installation page](https://stenv.readthedocs.io/en/latest/getting_started.html). 
```
